openedx/frontend-app-ecommerce
# chore: backport upgrade to node 18

## Description

Backport the following changes:

https://github.com/openedx/frontend-app-ecommerce/pull/274
https://github.com/openedx/frontend-app-ecommerce/pull/278

These backports are necessary to build the microfrontend apps of the ecommerce application in the open-release/palm.master branch with node 18 (and not node 16).

## Supporting information

Issue was raised here: https://github.com/openedx/wg-build-test-release/issues/261#issuecomment-1584746372

## Testing instructions

Check that this version builds with the "palm" branch of the tutor-ecommerce plugin: https://github.com/overhangio/tutor-ecommerce/pull/42
